doc/exts/pylint_extensions.py

```diff
@@ -2,7 +2,8 @@
 # Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
 # For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
 
-"""Script used to generate the extensions file before building the actual documentation."""
+"""Script used to generate the extensions file before building the actual
+documentation."""
```

Comment:
This doesn't actually enforce PEP8. See [here](https://www.python.org/dev/peps/pep-0008/#id31).
Perhaps there is a setting to toggle, but I would like a checker that at least does this 😄

Reply:
I'm going to apply the good modification for a start, and maybe fix the tool at this source (temporary fork ?). It's the only autofixer I found.

Reply:
Tools seems to be making quite a bit of unnecessary and arbitrary changes.
Somebody should probably take a good look at all suggested changes.

I would not put the tool in `pre-commit` for now. It doesn't really do what we're looking for.
Perhaps I'll work on something like this a hobby project 😄

Reply:
Sure it's not usable in the current form. There are some strange options. One to always line break before and after string, one only before. I guess I could fix that to line break when a single line would not work. Also I'd like to add capitalization for the first letter, probably not that hard. All in all I think the tool is fixable and I'm going to try it next year 😄 .